tests/experiments/registry_test.py

```diff
@@ -189,3 +189,55 @@ def test_default_initializer_is_first_in_list(self):
         Registry.default_initializer = "orthogonal"
         assert Registry.list_initializers()[0] == "orthogonal"
         Registry.default_initializer = default_initializer
+
+    # Seq2Seq encoders
+
+    def test_registry_has_builtin_seq2seq_encoders(self):
+        # pylint: disable=protected-access
+        from allennlp.modules.seq2seq_encoders.wrapped_pytorch_rnn import WrappedPytorchRnn
```

Comment:
I was checking that it's actually what we think it is.  I guess the querying `_module_class` on the object is probably good enough...